test/src/main/java/org/jvnet/hudson/test/JenkinsRule.java

```diff
@@ -1690,7 +1691,11 @@ void resolveDependencies() throws Exception {
 
                             File dst = new File(home, "plugins/" + artifactId + ".jpi");
                             if(!dst.exists() || dst.lastModified()!=dependencyJar.lastModified()) {
-                                FileUtils.copyFile(dependencyJar, dst);
+                                try {
+                                    FileUtils.copyFile(dependencyJar, dst);
+                                } catch (ClosedByInterruptException x) {
+                                    throw new AssumptionViolatedException("copying dependencies was interrupted", x);
```

Comment:
Perhaps it is worth to determine the root cause and fix it instead of eating the error.